src/webchat-ui/components/presentational/chat-options/RatingWidget.tsx

```diff
@@ -0,0 +1,150 @@
+import React from "react";
+import styled from "@emotion/styled";
+import { Typography } from "@cognigy/chat-components";
+import RatingDown from "../../../assets/rating-down-16px.svg";
+import RatingUp from "../../../assets/rating-up-16px.svg";
+import IconButton from "../IconButton";
+import PrimaryButton from "../PrimaryButton";
+import { createNotification } from "../Notifications";
+import MultilineInput from "../MultilineInput";
+
+const RatingWidgetRoot = styled.div(() => ({
+	width: "100%",
+	height: "310px",
+	display: "flex",
+	flexDirection: "column",
+	padding: "20px 0",
+	gap: 24,
+}));
+
+const RatingButtonContainer = styled.div(() => ({
+	width: "100%",
+	display: "flex",
+	alignItems: "flex-start",
+	gap: 8,
+}));
+
+const RatingButton = styled(IconButton)(({ theme, selected }) => ({
+	background: selected ? theme.primaryColor : theme.black95,
+	display: "flex",
+	padding: "12px 16px",
+	justifyContent: "center",
+	alignItems: "center",
+	gap: 16,
+	borderRadius: 15,
+	"svg": {
+		width: 16,
+		height: 16,
+	},
+	"&:focus": {
+		outline: "none",
+		boxShadow: `inset 0 0 0 2px ${theme.primaryColor}`,
+	},
+}));
+
+const RatingUpIcon = styled(RatingUp)(({ theme, selected }) => ({
+	"path": {
+		fill: selected ? theme.black95: theme.black10,
+	}
+}));
+
+const RatingDownIcon = styled(RatingDown)(({ theme, selected }) => ({
+	"path": {
+		fill: selected ? theme.black95: theme.black10,
+	}
+}));
+
+const RatingTextContainer = styled.div(() => ({
+	display: "flex",
+	flexDirection: "column",
+	justifyContent: "center",
+	alignItems: "flex-start",
+	gap: 16,
+	alignSelf: "stretch",
+}));
+
+const SendButton = styled(PrimaryButton)(({ theme }) => ({
+	width: "100%",
+	"&:disabled": {
+		opacity: 1,
+	},
+	"&:focus": {
+		outline: "none",
+		border: `2px solid ${theme.primaryColor}`,
+	},
+}));
+
+type TRatingValue = 1 | -1 | null;
+interface IOnSendRatingProps {
+	rating: TRatingValue;
+	comment: string;
+}
+interface IRatingWidgetProps {
+	ratingTitleText: string;
+	ratingCommentText: string;
+	onSendRating: (props: IOnSendRatingProps) => void;
+}
+
+export const RatingWidget = (props: IRatingWidgetProps) => {
+	const { ratingTitleText, ratingCommentText, onSendRating } = props;
+	const [ratingValue, setRatingValue] = React.useState<TRatingValue>(null);
+	const [ratingText, setRatingText] = React.useState("");
+
+	const ratingInputRef = React.useRef(null);
+	const sendRatingButtonRef = React.useRef(null);
```

Comment:
By code style convention, we usually import hooks and use them directly without `React.` default import

Reply:
Done